.circleci/config.yml

```diff
@@ -18,6 +18,11 @@ configure_step: &configure_step
     name: Configure
     command: mkdir build && cd build && cmake -GNinja -DCMAKE_TOOLCHAIN_FILE=../Support/CMake/Toolchain-${TARGET-${CIRCLE_JOB}}.cmake ..
 
+configure_android16_step: &configure_android16_step
```

Comment:
Instead of writing new rules for android API 16, you should probably add support for an env like "EXTRA_CMAKE_FLAGS" in `configure_step` so you don't have to duplicate the rules.